README.md

```diff
@@ -3,6 +3,8 @@
 ![user-data-for-fraud-prevention logo](./user-data-for-fraud-prevention-logo.png)
 
 [![CircleCI](https://circleci.com/gh/intuit/user-data-for-fraud-prevention/tree/master.svg?style=shield)](https://circleci.com/gh/intuit/user-data-for-fraud-prevention/tree/master)
+[![Coverage Status](https://coveralls.io/repos/github/intuit/user-data-for-fraud-prevention/badge.svg?branch=ci-coverage)](https://coveralls.io/github/intuit/user-data-for-fraud-prevention?branch=ci-coverage)
+![NPM Version](https://img.shields.io/npm/v/user-data-for-fraud-prevention)
```

Comment:
I was checking on your branch and coverage shows an unknown. The CI build failed on master branch after merging the previous PR.

Reply:
![Uploading Screen Shot 2020-10-05 at 14.13.50.png…]()

Reply:
Yes I just saw it. Have you added the key to circle ci environment variables ?

Reply:
COVERALLS_REPO_TOKEN

Reply:
Yes..I did. 
![Uploading Screen Shot 2020-10-05 at 14.18.03.png…]()

Reply:
Looks like the error is due to this:
![Uploading Screen Shot 2020-10-05 at 14.13.50.png…]()

Reply:
Ran all test suites.

/home/circleci/user-data-for-fraud-prevention/node_modules/coveralls/bin/coveralls.js:19
      throw err;
      ^
Bad response: 422 {"message":"Couldn't find a repository matching this job.","error":true}
npm ERR! code ELIFECYCLE
npm ERR! errno 1
npm ERR! user-data-for-fraud-prevention@1.0.2 test:coverage: `npm test -- --coverage && cat ./coverage/lcov.info | coveralls`
npm ERR! Exit status 1
npm ERR! 
npm ERR! Failed at the user-data-for-fraud-prevention@1.0.2 test:coverage script.
npm ERR! This is probably not a problem with npm. There is likely additional logging output above.

npm ERR! A complete log of this run can be found in:
npm ERR!     /home/circleci/.npm/_logs/2020-10-05T12_07_13_971Z-debug.log


Exited with code exit status 1
CircleCI received exit code 1

Reply:
![Screenshot 2020-10-05 at 6 51 37 PM](https://user-images.githubusercontent.com/20852629/95084659-cedd3300-073b-11eb-8bef-7ed29b84ff5c.png)

You can take a look at my badges-in-readme branch

Reply:
@skodamarthi  Im unable to see the screenshots

Reply:
`Bad response: 422 {"message":"Couldn't find a repository matching this job.","error":true}` this is because it's unable to find the token

Reply:
@christyjacob4 Here is the current screenshot from your branch as the circleCI is failing.
<img width="1137" alt="Screen Shot 2020-10-05 at 14 23 56" src="https://user-images.githubusercontent.com/4538858/95085029-91b27a00-0716-11eb-8ed7-22a0e3614f92.png">

Reply:
<img width="1402" alt="Screen Shot 2020-10-05 at 14 13 50" src="https://user-images.githubusercontent.com/4538858/95085129-b870b080-0716-11eb-87c3-8b180bd69ec7.png">

Reply:
I initially generated the token in coveralls and added the same as part of project settings in circleCI. Not sure if some local configuration is also needed in the repo.

Reply:
Yea It shows as failing because Im referencing `intuit/user-data-for-fraud-prevention` in the readme
On my fork, I have setup circle ci as well. Everything is running perfectly there. 
![Screenshot 2020-10-05 at 6 58 43 PM](https://user-images.githubusercontent.com/20852629/95085399-cd603a80-073c-11eb-91df-a06dbf94267f.png)

Reply:
No local configs are required